manage.py

```diff
@@ -1041,6 +1043,11 @@ def _add_terraform_subparser(subparsers):
         help=ARGPARSE_SUPPRESS,
         nargs='+')
 
+    tf_parser.add_argument(
+        '--cluster',
```

Comment:
consider making this flag plural (`--clusters`) since it can take 1 or more args